src/utils/wallet.js

```diff
@@ -665,7 +666,7 @@ class Wallet {
 
     async recoverAccountSeedPhrase(seedPhrase, accountId, fromSeedPhraseRecovery = true) {
         const { publicKey, secretKey } = parseSeedPhrase(seedPhrase)
-        const accountIds = await getAccountIds(publicKey)
```

Comment:
seems to me that it still makes sense to wrap logic into `getAccountIds` vs using `sendJson` directly